### ISSUE TITLE
Android: can't build Demo.Droid: Error retrieving parent for item: No resource found that matches the given name

### ISSUE THREAD

**Issue by wtheronjones** [rotorgames/Rg.Plugins.Popup#232]
I downloaded the source and I can't build the Demo.Droid project.

I am getting this error, using Visual Studio 2017:
Error		Error retrieving parent for item: No resource found that matches the given name 'android:TextAppearance.Material.Widget.Button.Colored'.	Rg.Plugins.Popup.Droid	C:\Projects\xamarin\Rg.Plugins.Popup\src\Rg.Plugins.Popup.Droid\obj\Debug\__library_projects__\Xamarin.Android.Support.v7.AppCompat\library_project_imports\res\values-v24\values-v24.xml	

If I need a specific Android SDK or build tools installed, I have Android SDK Tools 25.2.5 and Android SDK Platform Tools 25.0.5 installed.

I tried commenting out these two elements in the values-v24.xml file:

```
<style name="Base.TextAppearance.AppCompat.Widget.Button.Borderless.Colored"
         parent="android:TextAppearance.Material.Widget.Button.Borderless.Colored" />
  <style name="Base.TextAppearance.AppCompat.Widget.Button.Colored"
         parent="android:TextAppearance.Material.Widget.Button.Colored" />
```

And that leads to the next error:  No resource identifier found for attribute 'scrollIndicators in package 'android', in file C:\Projects\xamarin\Rg.Plugins.Popup\src\Demo\Demo.Droid\obj\Debug\__library_projects__\Xamarin.Android.Support.v7.AppCompat\library_project_imports\res\layout\abc_alert_dialog_button_bar_material.xml

At this point, I stopped.

any tips?

Great work on this library, by the way!

**Comment by rotorgames** [comment]
@wtheronjones What versions of Xamarin.Android, Xamarin.Forms, Visual Studio, Windows / Mac OS do you use?

**Comment by wtheronjones** [comment]
@rotorgames Here is the version info:
Windows 10
Visual Studio Pro 2017 (15.3.3).  
Xamarin.Android SDK 7.4.0.21 
Xamarin.Forms is 2.5.0.121934

The source code is from the master branch as of yesterday.

The full path to the file w/ the errors is located in: C:\Projects\xamarin\Rg.Plugins.Popup\src\Rg.Plugins.Popup.Droid\obj\Debug\__library_projects__\Xamarin.Android.Support.v7.AppCompat\library_project_imports\res\values-v24\values-v24.xml

The version of Xamarin.Android.Support.v7.AppCompat is 25.4.0.2

**Comment by rotorgames** [comment]
@wtheronjones Is this problem only in the demo project? Does this plugin work from nuget?

**Comment by wtheronjones** [comment]
@rotorgames  I finally got things going.  Most of the drama wasn't due to the Popup library, but due to the dependencies on the Android Support libraries required by Xamarin.Forms on Android vs the current Android Target Framework.

This combination of libraries works in VS 2015:
Android SDK 26 or 27(Android 7.0 or 8.0)
"MyForms.csproj" - a Xamarin Forms Portable class library that I converted to target .netstandard 1.0
   - references NETStandard.Library, Microsoft.NETCore.Portable.Compatibility
Rg.Plugins.Popup 1.1.4.135-beta
Xamarin.Forms 2.5.0.121934
Xamarin Android Compatibility libraries version 25.4.0.2
 
Note that the compatibility libraries installed by default are 23.3.0. When that was the case, I got this error: 
"Could not load assembly 'Xamarin.Android.Support.Fragment, Version=1.0.0.0, Culture=neutral, PublicKeyToken='. Perhaps it doesn't exist in the Mono for Android profile?"
To fix that I updated the compatibility libraries to 25.4.0.2, but this version seems to have a bug in it. it puts the right version on disk, but updates the csproj to reference the files via the wrong version #.  I had to edit the csproj near the bottom to remove the errors.   It's the part at the bottom of the .csproj w/ the Target Name="EnsureNuGetPackageBuildImports" and as Error tasks to check for the existence of a file.
